### PR TITLE
Add `vttablet` flags from vitessio/vitess#13690

### PR DESCRIPTION
**Pull request by timvaillancourt**
This PR adds new `vttablet` flags that are added in https://github.com/vitessio/vitess/pull/13690

### CODE REVIEW

File: content/en/docs/18.0/reference/programs/vttablet.md [hunk 1]
```diff
@@ -402,8 +402,10 @@ The following global options apply to `vttablet`:
...
+| --tx-throttler-topo-refresh-interval                             | duration      | The rate that the transaction throttler will refresh the topology to find cells. (default 5m0s)                                                                                                                                                                                        |
+| --tx-throttler-tx-pool-thresholds                                | float:float   | The low:high tx pool usage percentages for the transaction throttler to begin throttling low-priority transactions, defined as two colon-separated floats.                                                                                                                                                                                                                                                                                                                                                 |
```
Comment (shlomi-noach) [inline content/en/docs/18.0/reference/programs/vttablet.md:408]: Shall we clarify that these are non-SELECTs?
```suggestion
| --tx-throttler-tx-pool-thresholds                                | float:float   | The low:high tx pool usage percentages for the transaction throttler to begin throttling low-priority transactions (`INSERT/UPDATE/DELETE`), defined as two colon-separated floats.                                                                                                                                                                                                                                                                                                                                                 |
```

File: content/en/docs/18.0/reference/programs/vttablet.md [hunk 2]
```diff
@@ -402,8 +402,10 @@ The following global options apply to `vttablet`:
...
 | --tx-throttler-healthcheck-cells                                 | strings       | Synonym to -tx_throttler_healthcheck_cells                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| --tx-throttler-query-pool-thresholds                             | float:float   | The low:high query pool usage percentages for the transaction throttler to begin throttling low-priority select queries, defined as two colon-separated floats.                                                                                                                                                                                                                                                                                                                                            |
```
Comment (shlomi-noach) [inline content/en/docs/18.0/reference/programs/vttablet.md:405]:
```suggestion
| --tx-throttler-query-pool-thresholds                             | float:float   | The low:high query pool usage percentages for the transaction throttler to begin throttling low-priority `SELECT` queries, defined as two colon-separated floats.                                                                                                                                                                                                                                                                                                                                            |
```

**Comment by frouioui** [comment]
@ajm188 is this still relevant considering the recent flag documentation automation that has been going on?

**Comment by ajm188** [comment]
nah this should get picked up automatically

**Comment by frouioui** [comment]
@timvaillancourt I am closing this for now as the new Cobra documentation should automatically pick this change as mentioned by @ajm188, we can re-open it if needed.

**Pull request closed without merging**